sources/dictionary/resource.yml

```diff
@@ -85,6 +85,7 @@ tabularDataResource:
   properties:
     profile:
       "$ref": "#/definitions/profile"
+            enum: ["tabular-data-resource"]
```

Comment:
This indentation breaks `npm run compile`

Reply:
@roll thanks. fixed in https://github.com/frictionlessdata/specs/pull/515/commits/9dd18d6541d4d3d7ea2b69bfdc66bd915e1defc5